test/crash/src/test/java/io/narayana/lra/arquillian/LRACoordinatorRecoveryIT.java

```diff
@@ -150,57 +151,24 @@ public static WebArchive deploy() {
     public void lraCoordinatorShortTimeoutLRA(@ArquillianResource @OperateOnDeployment(LRA_PARTICIPANT_DEPLOYMENT_QUALIFIER) URL baseURL)
             throws URISyntaxException {
 
-        String lraId;
-        URI lraListenerURI = UriBuilder.fromUri(baseURL.toURI()).path(LRAListener.LRA_LISTENER_PATH).build();
-
-        // Starts an LRA with a short time limit by invoking a resource annotated with @LRA. The time limit is
-        // defined as LRAListener.LRA_SHORT_TIMELIMIT
-        Response response = client
-                .target(lraListenerURI)
-                .path(LRAListener.LRA_LISTENER_ACTION)
-                .request()
-                .put(null);
-
-        Assert.assertEquals("LRA participant action", 200, response.getStatus());
+        Long SHORTER_TIMEOUT = 1000L;
```

Comment:
Using here a shorter timeout (1 sec) so that the shortLRA times out when the server is down.